ts/sagas/startup/checkSupportedFingerprintSaga.ts

```diff
@@ -0,0 +1,60 @@
+import { Effect } from "redux-saga";
+import { call, put, take } from "redux-saga/effects";
+import { getType } from "typesafe-actions";
+
+import { navigateToOnboardingFingerprintScreenAction } from "../../store/actions/navigation";
+import { fingerprintAcknowledge } from "../../store/actions/onboarding";
+
+import TouchID, { IsSupportedConfig } from "react-native-touch-id";
+
+export type BiometrySimpleType =
+  | "Fingerprint"
```

Comment:
use all caps, snake case names for "kind" constants (i.e. `FINGERPRINT`, `NOT_ENROLLED`, `UNAVAILABLE`)

Reply:
Switched to all-caps BiometryType